app/uk/gov/hmrc/nisp/views/main.scala.html

```diff
@@ -46,6 +47,9 @@
   <li><a href="@Constants.logoutUrl" role="button">Sign out</a></li>
 }
 
+@contentWithTitle =  {
+    @mainContent
```

Comment:
Trying adding `@pageTitle` to here. It already seems to be setup to expand the article section.